os/arch/arm/src/armv7-a/arm_prefetchabort.c

```diff
@@ -152,6 +152,8 @@ uint32_t *arm_prefetchabort(uint32_t *regs, uint32_t ifar, uint32_t ifsr)
 		CURRENT_REGS = savestate;
 	} else {
 		if (!IS_SECURE_STATE()) {
+			/* Add new line to distinguish between normal log and assert log.*/
+			lldbg_noarg("\n");
```

Comment:
How about adding `\n` at the start of the first message in the print_prefecthabort_detail?
I mean line 79.